src/molecules/form-fields/select-form-field.test.tsx

```diff
@@ -1,9 +1,171 @@
-// import React from "react";
-// import { render } from "@testing-library/react";
-// import { SelectFormField } from "./select-form-field";
-// import faker from "faker";
-// import uuid from "uuid";
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import {
+    SelectFormField,
+    InvalidSelectFormValueClass,
+} from "./select-form-field";
+import faker from "faker";
+import uuid from "uuid";
 
 describe("SelectFormField", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/26", () => {});
+    test("when default props, renders SelectFormField with label", () => {
```

Comment:
suggestion: remove 'SelectFormField' from test names as it is inferred and automatically added by outermost test fixture name of "SelectFormField"